src/Content/Smilies.php

```diff
@@ -285,4 +285,17 @@ private static function heartReplaceCallback(array $matches): string
 
 		return str_replace($matches[0], $t, $matches[0]);
 	}
+
+	/**
+	 * Checks if the body only contains 4 byte unicode characters.
+	 *
+	 * @param string $body
+	 * @return boolean
+	 */
+	public static function isEmojiPost(string $body): bool
+	{
+		$conv = html_entity_decode(preg_replace('#\s#', '', $body));
```

Comment:
This will still be stunted by `&nbsp;` entities that should be decoded first before running through `preg_replace`.
```suggestion
		$conv = preg_replace('#\s#', '', html_entity_decode$body));
```

Reply:
Done